bears/js/ESLintBear.py

```diff
@@ -58,15 +58,31 @@ def generate_config(filename, file):
         return '{"extends": "eslint:recommended"}'
 
     def process_output(self, output, filename, file):
-        if output[1]:
+        def warn_issue(message):
             self.warn('While running {0}, some issues were found:'
                       .format(self.__class__.__name__))
-            self.warn(output[1])
+            self.warn(message)
 
-        if not file or not output[0]:
+        # Taking output from stderr for ESLint 2 program errors.
+        # ESLint 2 is no longer supported, but it is here so that anyone
+        # with ESLint 2 will still see any errors that were emitted.
+        if output[1]:  # pragma: no cover
+            warn_issue(output[1])
+
+        output = output[0]
```

Comment:
Can you remove this line.
If you keep using `output[0]` , the diff will be smaller, and git will recognise that `output = json.loads(output[0])` was indented.